lncfg/defaults.go

```diff
@@ -0,0 +1,14 @@
+package lncfg
+
+import (
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+const (
+	// DefaultBitcoinTimeLockDelta Default TLD for channel policy updates
```

Comment:
Should be a full sentence, for example:
`DefaultBitcoinTimeLockDelta is the default TLD for channel policy updates.`
Applies to the other comments too.